src/libs/actions/Policy.js

```diff
@@ -83,6 +90,14 @@ function getSimplifiedPolicyObject(fullPolicyOrPolicySummary, isFromFullPolicy)
     };
 }
 
+/**
+ * Stores in Onyx the policy ID of the last workspace that was accessed by the user
+ * @param {String} policyID
```

Comment:
I think the param type here is now `@param {String|null} policyID`. Can you update?

Reply:
@chiragsalian @hellohublot I was wondering if we should set this to empty string `''` instead of `null`? 

What do you folks think?

Reply:
I don't have a strong preference as long as they both work. Looking at the code in RequestCallPage I believe it would work with either. So with that said I think setting to null is fine with me.